libass/ass.c

```diff
@@ -1528,11 +1528,16 @@ int ass_track_set_feature(ASS_Track *track, ASS_Feature feature, int enable)
 #ifdef USE_FRIBIDI_EX_API
     case ASS_FEATURE_BIDI_BRACKETS:
         track->parser_priv->bidi_brackets = !!enable;
+        break;
```

Comment:
This may or may not be the right thing -- I'm not entirely sure what the expected behaviour is here currently: is `ASS_FEATURE_INCOMPATIBLE_EXTENSIONS` expected to enable all incompatible features using fallthrough?

Reply:
(and if so, one supposes the strategy needs some adjusting, since one presumably has to also be able to individually select features, which fallthrough would preclude)

Reply:
It is just expected to enable all incompatible features. The fallthrough presumably has to be replaced by a list (and feel free to smack me for micro-optimizing using a fallthrough in the first place). Compare with: https://github.com/libass/libass/pull/441/files.

Reply:
Sounds good, would you like that in a separate commit or in the same one?